ports/zephyr/boards/manifest_xiao_ble_nrf52840_sense.py

```diff
@@ -0,0 +1,11 @@
+# This is an example frozen module manifest. Enable this by configuring
+# the Zephyr project and enabling the frozen modules config feature.
```

Comment:
I think this comment can be removed, it's not really an example anymore for this board.